roles/dbid-logger/tasks/main.yml

```diff
@@ -30,10 +32,10 @@
 
 - name: Extract DBID from query result
   ansible.builtin.set_fact:
-    oracle_dbid: "{{ dbid_query_result.stdout_lines | select('match', '^\\d+$') | first | default('') | trim }}"
```

Comment:
It still doesn't hurt to trim here, especially in terms of trailing spaces.

Reply:
Also, all the sql*plus formatting still doesn't _guarantee_ that there won't be trailing whitespace.  Just to be conservative, consider a regex like `^\\w*\\d+\\w*$` that will match leading and trailing whitespace (that the trim command will then strip out).